plugins/services/src/js/validators/PlacementsValidators.js

```diff
@@ -18,7 +18,7 @@ function checkDuplicateOperatorField(constraints) {
 
     if (visitedOperatorFieldPairs.indexOf(key) !== -1) {
       errors.push(
-        [].concat([
```

Comment:
We could als do `return [...errors, {...}, {...}]`, I think that would be a bit easier to read 🤔